testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1455,7 +1455,7 @@
 When(/^I run spacewalk-hostname-rename command on the server$/) do
   temp_server = twopence_init("ssh:#{$server.public_ip}")
   temp_server.extend(LavandaBasic)
-  command = "spacewalk-hostname-rename #{$server.public_ip}
+  command = "echo admin | spacewalk-hostname-rename #{$server.public_ip}
```

Comment:
Do we only need a password here and no user?

Reply:
It's just that `spacecmd` asks for it and uses the cached user

Reply:
Aha. Thank you Cedric.